en/docs/develop/oauth2-scope-management-rest-apis.md

```diff
@@ -1,24 +1,31 @@
 ---
 template: templates/swagger.html
 ---
-# OAuth 2.0 Scope Endpoint Rest API Definition - v1
 
-??? Note "Note: Click to view"
-    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server.
+# OAuth 2.0 Scope Management Rest API Definition - v1
 
-    1.  Expand the relevant API operation and click the **Try It Out** button.
-    2.  Fill in relevant sample values for the input parameters and click **Execute**.
-        You will receive a sample curl command with the sample values you filled in.
-    3. Add a `-k` header to the curl command and run the curl command on the terminal with a running instance of WSO2 IS.
+The OAuth2 scope API in WSO2 Identity Server can be used to manage oauth2 scopes and scope bindings(role, permission 
```

Comment:
Identity Server (IS) 

oauth2 scopes and scope bindings such as roles and permissions.